CONTRIBUTING.md

```diff
@@ -77,11 +77,10 @@ $ git clone git@github.org:<github_username>/sceptre.git
 3. Install Sceptre for development (we recommend you use a
    [virtual environment](http://docs.python-guide.org/en/latest/dev/virtualenvs/))
 
+A convenient script is also provided:
```

Comment:
no, we are going to use poetry and the poetry CLI to manage dependencies very soon, check PR #1323

Reply:
Will remove if the other discussions are resolved.